docs/data/joy/components/button/button.md

```diff
@@ -33,10 +33,14 @@ export default function MyApp() {
 ### Variants
 
 The button component supports the four global variants: `solid` (default), `soft`, `outlined` and `plain`.
-Choose one of them depending on the button's action importance.
+Choose one of them depending on the button's [action importance](/joy-ui/main-features/global-variants/#hierarchy-of-importance).
 
 {{"demo": "ButtonVariants.js"}}
 
+:::info
+To add more variants to the component, check out the [customization](/joy-ui/customization/themed-components/#extend-variants) section.
+:::
+
```

Comment:
@danilo-leal What do you think about adding this info to all components?

Reply:
```suggestion
:::info
💡**Learn:** To add more variants to the component, visit  the [extended variants](/joy-ui/customization/themed-components/#extend-variants) section on the Customization page.
:::
```

I dig it! I think it might be a good idea.

Reply:
using `visit` is a lot better. I think it should be consistent throughout the docs. cc @samuelsycamore

Reply:
I'm cool with either term. I like to change it up throughout the docs so it's not too repetitive.

Reply:
One thought about using `💡**Learn:**` inside of the `:::info` callout—what if we replaced them both with a `:::success` callout instead? We don't really use this green callout much in the docs yet. We could repurpose it to be used for helpful tips and guidance like this. We would ideally train devs to look for the green callouts for more info/learning resources. What do you think?

Reply:
Totally onboard with it ☝️